pkg/controller/vsphere/actuator_test.go

```diff
@@ -96,7 +96,7 @@ func TestMachineEvents(t *testing.T) {
 
 	configNamespace := &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: openshiftConfigNamespace,
+			Name: getOpenshiftConfigNamespace(),
```

Comment:
All of the tests can use a constant, we need the code to work against some function or injected value